kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -47,6 +47,7 @@ function _contentState(data) {
   } else {
     progress = data.progress_fraction;
   }
+  const assessmentMetaData = data.asessmentmetadata || {};
```

Comment:
`[0]` won't work if/when the `[]` option is used.

There's also some oddness going on below:

```javascript
    assessmentIds: JSON.parse(assessmentMetaData.assesment_item_ids || '[]'),
    masteryModel: JSON.parse(assessmentMetaData.mastery_model || '{}'),
    randomize: assessmentMetaData.randomize || false,
```

Seems unusual for the API to be returning json strings instead of objects and to have so many conditionally-available values?